modules/serverless-resolving-operator-upgrade-failure.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * /serverless/admin_guide/serverless-upgrades.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-resolving-operator-upgrade-failure_{context}"]
+= Resolving {ServerlessOperatorName} upgrade failure
+
+In some cases, such as when you do manual uninstalls and reinstalls, you might encounter an error when upgrading {ServerlessOperatorName}. In this situation, you need to manually reinstall {ServerlessOperatorName}.
+
+.Procedure
+
+. Identify the version of {ServerlessOperatorName} that was installed originally using the `project.yaml` files for the different releases.
++
+For example, the error message during attempted upgrade contains this string:
++
+[source,yaml]
+----
+The installed KnativeServing version is v0.25.1.
+----
++
+In this case, the KnativeServing version is 0.25.1, which corresponds to {ServerlessOperatorName} 1.19.
+
+. Uninstall {ServerlessOperatorName} and all its install plans.
```

Comment:
For this step, I'd point to [this section](https://docs.openshift.com/container-platform/4.12/serverless/removing/removing-serverless-operator.html#olm-deleting-operators-from-a-cluster-using-web-console_removing-serverless-operator) which does exactly what they need: remove subscription, installplans and serverless-operator ClusterServiceVersion.